symphony-bdk-http/symphony-bdk-http-jersey2/build.gradle

```diff
@@ -19,6 +19,7 @@ dependencies {
     implementation 'commons-io:commons-io'
     implementation 'org.apache.commons:commons-lang3'
     implementation 'org.glassfish.jersey.connectors:jersey-apache-connector'
+    implementation 'commons-logging:commons-logging'
```

Comment:
why do we need it?

Reply:
It is a dependency of jersey-apache-connector, see https://eclipse-ee4j.github.io/jersey.github.io/project-info/2.32/jersey/project/jersey-apache-connector/dependencies.html
So when we are using the `ApacheConnectorProvider`, `AuthMain` fails with `java.lang.NoClassDefFoundError: org/apache/commons/logging/LogFactory`. I can change that to runtimeOnly

Reply:
Ok, I could not figure out why it it not come up as a transitive dependency there ... maybe add a comment either inline or the git log explaining that is required by jersey apache connector